bin/pit-init.sh

```diff
@@ -56,21 +57,24 @@ if [ -e /dev/disk/by-label/PITDATA ]; then
     mount -v -L PITDATA
 fi
 
-PREP_DIR=/var/www/ephemeral/prep
-CSI_CONF=$PREP_DIR/system_config.yaml
+PREP_DIR=${PITDATA}/prep
+DATA_DIR=${PITDATA}/data
+CONF_DIR=${PITDATA}/configs
+[ ! -d $PREP_DIR ] && mkdir -pv $PREP_DIR
+[ ! -d $DATA_DIR ] && mkdir -pv $DATA_DIR
+[ ! -d $CONF_DIR ] && mkdir -pv $CONF_DIR
+
+
 set -e
-if [ ! -d $PREP_DIR ] ; then
-    echo >&2 "Need $PREP_DIR; this needs to be created by hand if this is a first-time install."
-    echo >&2 "See: https://github.com/Cray-HPE/docs-csm/blob/main/install/bootstrap_livecd_usb.md#configuration-payload"
-    exit 1
-elif [ ! -f $CSI_CONF ] ; then
+CSI_CONF=$PREP_DIR/system_config.yaml
+if [ ! -f $CSI_CONF ] ; then
     echo >&2 'CSI needs inputs; no $CSI_CONF file detected!'
     echo >&2 'See: https://github.com/Cray-HPE/docs-csm/blob/main/install/bootstrap_livecd_usb.md#generate-installation-files'
     exit 1
```

Comment:
Should we be linking the remote iso version here as well / instead of?

Reply:
Also, this links to `main`, which is probably not a super great idea.

Reply:
This actually should point to `prepare_configuration_payload.md`

Reply:
It would have to fancily parse the CSM_PATH to resolve that, but if a user ends at `main` they can (and should) be able to select the branch they actually need.

Reply:
Fixed the URL to point to the right document and added a TODO for MTL-1695. If you have an idea for autoresolving the branch I'm all for it.